src/App.test.tsx

```diff
@@ -14,10 +14,10 @@ describe('<App />', () => {
   });
 
   it('contains NavContainer', async () => {
-    const { queryByType } = render(<App />);
+    const { UNSAFE_getByType } = render(<App />);
```

Comment:
`queryBy*` and `getBy*` commands are a little bit different if I remember it correctly. Isn't there `UNSAFE_queryByType` method?